src/main/scala/util/package.scala

```diff
@@ -57,4 +65,25 @@ package object util {
     println(s"${msg}: ${minutesStr}${secondsStr}${millisStr}")
     result
   }
+
+  def dotPrintTmp(
+    name: String,
+    r: elevate.core.RewriteResult[elevate.rise.Rise]
+  ): Unit = r match {
+    case elevate.core.Success(p) => dotPrintTmp(name, p)
+    case _ =>
+  }
+
+  def dotPrintTmp(name: String, e: rise.core.Expr): Unit = {
+    import scala.language.postfixOps
+    import scala.sys.process._
+
+    val path = "/tmp"
```

Comment:
Why don't you use the `writeToTempFile` function that uses the `createTempFile` function and avoids hardcoding the `/tmp` path (that e.g. will not work on Windows).

Reply:
I can use `File.createTempFile` but it should not be deleted on exit, since this is used for debug.

Reply:
Maybe it is better if I provide a path parameter and set `/tmp` as the default?

Reply:
Well than use directly the `File.createTempFile(prefix, suffix)` method from Java.io.File without calling the `deleteOnExit` method.